pages/data_life_cycle/planning.md

```diff
@@ -18,8 +18,8 @@ contributors: [Dr. Nami Sunami, Dr. Lizette Guzman-Ramirez]
 All research projects need a data management plan, in some cases, depending on the data collected or the way the data is collected, you might need a privacy assessment and an ethical approval before you start your research. 
 
 - [Data management plan](dmp)
-- Privacy assessment <!-- Link to the new page with info about privacy -->
-- Ethical applications, when do I need an ethical approval? <!-- Link to the new page with the decision tree for ethical applications -->
+- [Privacy assessment](https://my.eur.nl/en/eur-employee/work-support/privacy/research-privacy) 
```

Comment:
I noticed that this will link out to a page that requires log-in. So I'm putting the lock icon here. ✌️ 

```suggestion
- [Privacy assessment<sup><i class="fa-solid fa-lock"></i></sup>](https://my.eur.nl/en/eur-employee/work-support/privacy/research-privacy)
```